### PR TITLE
Set up Behat

### PR DESCRIPTION
**Pull request by forevermatt**

### CODE REVIEW

**Comment by forevermatt** [comment]
@lvail or @wcjr, what do you think about this? I tried to phrase the Behat feature scenarios to capture the bigger picture of what each set of interactions (for an ad-hoc user sync) would be, based on our discussions.

File: application/composer.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "php": ">=7.0",
+    "roave/security-advisories": "dev-master"
+  },
+  "require-dev": {
+    "behat/behat": "^3.3"
+  },
+  "autoload": {
+    "psr-4": {
+      "Sil\\Idp\\IdSync\\Behat\\Context\\": "features/bootstrap"
+    }
```
Comment (wcjr) [inline application/composer.json]: I don't think you'll need this since it's referenced in the behat.yml.

**Comment by wcjr** [comment]
I thnk this works really well

File: application/composer.json (hunk 1 above)
Comment (forevermatt) [inline application/composer.json]: Hmm... you may be right about that

**Pull request merged**